00_basics/sc_dimensions.ipynb

```diff
@@ -0,0 +1,771 @@
+{
```

Comment:
<p>In Jupyter Notebook, we stop the audio playback of the SuperCollider server by executing a single dot.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/capital-G/musikinformatik-sose2021/pull/23/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>